### PR TITLE
feat(postgres): document PostgreSQL performance tuning

### PR DESCRIPTION
**Pull request by educhastenier**
Closes [BR-250](https://bonitasoft.atlassian.net/browse/BR-250)

### CODE REVIEW

**Review by pabloalonso — CHANGES_REQUESTED** [review]
Describe a Prod ready infraestructure like the one used for our Audit.
Use these values (specially RAM) to set up proper values on shared_buffers and effective_cache_size

File: md/performance-tuning.md [hunk 1]
```diff
@@ -487,3 +485,63 @@ Details on these properties can be found in [the Quartz documentation](http://ww
 
 They are not read subsequently, so changing the values in `bonita-tenant-community-custom.properties` after the Engine has been started has no effect on Quartz.
 For value definition, and information about how to update the Quartz trigger tables, see the [Quartz documentation](http://www.quartz-scheduler.org/documentation/) about Cron Triggers.
+
+
+#### PostgreSQL performance tuning
+
+Here is Bonita advice to finely tune PostgreSQL database server performance.
+
+In this example, we assume with have:
```
Comment (baptistemesta) [inline md/performance-tuning.md]:
```suggestion
In this example, we assume you have:
```

File: md/performance-tuning.md [hunk 2]
```diff
@@ -487,3 +485,63 @@ Details on these properties can be found in [the Quartz documentation](http://ww
...
+```
+
+::: warning
+Warning: properties `random_page_cost` and `seq_page_cost` should have values relative to each other thoroughly set, in order
+for PostgreSQL query planner to choose the right execution plan.  
+See PostgreSQL [Planner Cost Constants](https://www.postgresql.org/docs/11/runtime-config-query.html#RUNTIME-CONFIG-QUERY-CONSTANTS)
+for more details on how to set those values.
+:::
+
+If you want to be able to **restore live PITR backup** of the database, ensure archiving is activated:
```
Comment (baptistemesta) [inline md/performance-tuning.md]: We could tell what is `PITR` or at least only write in parenthesis the meaning of the acronym)

**Review by baptistemesta — CHANGES_REQUESTED** [review]
Great PR.
Only few things to change.
Also should we link to tools that can be used to help configuring that. e.g. a link to pgCluu

**Comment by educhastenier** [comment]
@baptistemesta Feel free to contribute. I don't know pgCluu of pgBadger enough to document them...

File: md/performance-tuning.md (hunk 2 above)
Comment (educhastenier) [reply md/performance-tuning.md]: :heavy_check_mark: done

**Comment by baptistemesta** [comment]
I think that's a great first step. I don't know either pgCluu enough.

**Pull request merged**